apps/minaintyg/src/store/errorMiddleware.ts

```diff
@@ -0,0 +1,22 @@
+import { randomUUID } from '@frontend/utils'
+import { isRejectedWithValue, Middleware, MiddlewareAPI, SerializedError } from '@reduxjs/toolkit'
+import { FetchBaseQueryError } from '@reduxjs/toolkit/dist/query'
+
+export type QueryError = (FetchBaseQueryError & { id?: string }) | (SerializedError & { id?: string })
+
+/**
+ * Error handling middleware
+ * https://redux-toolkit.js.org/rtk-query/usage/error-handling
+ */
+export const errorMiddleware: Middleware = (_: MiddlewareAPI) => (next) => (action) => {
+  if (isRejectedWithValue(action)) {
+    const id = randomUUID()
```

Comment:
@ChrilleSigerhed The error id is generated here and added to the error payload for every failed request, this is what is rendered as error.id on all the alerts :)

Reply:
Haha, awsome! :)